classes/WP_CLI/ActionScheduler_WPCLI_Scheduler_command.php

```diff
@@ -5,6 +5,37 @@
  */
 class ActionScheduler_WPCLI_Scheduler_command extends WP_CLI_Command {
 
+	/**
+	 * Force tables schema creation for Action Scheduler
+	 *
+	 * ## OPTIONS
+	 *
+	 * @param array $args Positional arguments.
+	 * @param array $assoc_args Keyed arguments.
+	 * @throws \WP_CLI\ExitException When an error occurs.
+	 *
+	 * @subcommand fix-schema
+	 */
+	public function fix_schema( $args, $assoc_args ) {
+		$schema_classes = array( 'ActionScheduler_LoggerSchema', 'ActionScheduler_StoreSchema' );
+
+		foreach ( $schema_classes as $classname ) {
+			if ( is_subclass_of( $classname, 'ActionScheduler_Abstract_Schema' ) ) {
+				$obj = new $classname();
+				$obj->init();
+				$obj->register_tables( true );
+
+				WP_CLI::success(
+					sprintf(
+						/* translators: %s refers to the schema name*/
+						__( 'Registered schema for %s', 'action-sheduler' ),
```

Comment:
```suggestion
						__( 'Registered schema for %s', 'action-sheduler' ),
```
```suggestion
						__( 'Registered schema for %s', 'action-scheduler' ),
```

Typo in the text domain.